Makefile

```diff
@@ -417,8 +417,13 @@ images: image bundle-image  ## Build operator and bundle images.
 .PHONY: images-extra
 images-extra: openscap-image e2e-content-images  ## Build the openscap and test content images.
 
+.PHONY: prebuild-check
+prebuild-check: generate fmt vet test-unit 
+# Make sure no changes are present in the working directory
+	@git diff --exit-code > /dev/null || (echo "Working directory is not clean. Check git status." && exit 1)
```

Comment:
So the main advantage here is that `prebuild-check` will fail if there are changes that forgot to run `generate`, it looks like?